src/main/java/gate/util/persistence/PersistenceManager.java

```diff
@@ -1466,6 +1455,20 @@ public boolean getValue() {
   
   
   static {
+
+    xstream = new XStream(new SunUnsafeReflectionProvider(new FieldDictionary(new XStream12FieldKeySorter())),
```

Comment:
SunUnsafe.. hope that does not cause a problem in java 11 runtime.

Reply:
Added to follow up list: https://jira.corp.docusign.com/confu/display/ML/POC%3A+Update+Gate